docs/source/interfaces/ISIS SANS v2.rst

```diff
@@ -602,6 +602,44 @@ Note that the settings are logically grouped by significant stages in the reduct
+|       |                          | are used by the reduction.                                                              |
++-------+--------------------------+-----------------------------------------------------------------------------------------+
+| **2** | **Centre Position HAB**  | The centre position of the high angle bank. The first coordinate is horizontal          |
+|       |                          | and the second vertical. These boxes are populated by the user file and the values here |
+|       |                          | are used by the reduction.                                                              | 
++-------+--------------------------+-----------------------------------------------------------------------------------------+
+| **3** | **Minimum radius limit** | The minimum radius of the region used to ascertain centre position.                     |
++-------+--------------------------+-----------------------------------------------------------------------------------------+
+| **4** | **Maximum radius limit** | The maximum radius of the region used to ascertain centre position.                     |
++-------+--------------------------+-----------------------------------------------------------------------------------------+
+| **5** | **Max iterations**       | The maximum number of iterations the algorithm will perform before concluding its       |
+|       |                          | search.                                                                                 |
++-------+--------------------------+-----------------------------------------------------------------------------------------+
+| **6** | **Tolerance**            | If the centre position moves by less than this in an iteration the algorithm will       |
+|       |                          | conclude its search.                                                                    |
++-------+--------------------------+-----------------------------------------------------------------------------------------+
+| **7** | **Left/Right**           | Controls whether the algorithm searches for the centre in the left/right direction.     |
++-------+--------------------------+-----------------------------------------------------------------------------------------+
+| **8** | **Up/Down**              | Controls whether the algorithm searches for the centre in the up/down direction.        |
++-------+--------------------------+-----------------------------------------------------------------------------------------+
+| **9** | **Run**                  | Runs the SANSBeamCentreFinder algorithm the boxes **1** and **2** are updated with new  |
+|       |                          | values upon completion.                                                                 |
++-------+--------------------------+-----------------------------------------------------------------------------------------+
```

Comment:
a general question. Do 7 and 8 refer to an actual algorithm or options in the GUI?

Reply:
They refer to checkboxes in the GUI.

Reply:
I would not use the word algorithm then. I know it is eventually used by the algorithm but a user may get confused between an interface and an algorithm. You could just say: Controls whether to search for the ...